docs/reference-manual/native-image/Bundles.md

```diff
@@ -297,6 +377,52 @@ $ native-image --bundle-apply=micronautguide-pgo-optimized.nib
 ```
 
+## Executing the bundled application
+
+As described later in [Bundle File Format](#bundle-file-format), a bundle file is a JAR file with a contained launcher for launching the bundled application.
+This means you can use a native image bundle with any JDK and execute it as a JAR file with `<jdk>/bin/java -jar [bundle-file.nib]`.
+The launcher uses the command line arguments stored in _run.json_ and adds all JAR files and folders in _input/classes/cp_ and _input/classes/p_ to the classpath and module path respectively.
+
+The launcher also comes with a separate command line interface described in its help text:
+```
+This native image bundle can be used to launch the bundled application.
+
+Usage: java -jar bundle-file [options] [bundle-application-options]
+
+where options include:
+
+    --with-native-image-agent[,update-bundle[=<new-bundle-name>]]
+                runs the application with a native-image-agent attached
+                'update-bundle' adds the agents output to the bundle-files classpath.
+                '=<new-bundle-name>' creates a new bundle with the agent output instead.
+                Note 'update-bundle' requires native-image to be installed
+
+    --container[=<container-tool>][,dockerfile=<Dockerfile>]
+                sets up a container image for execution and executes the bundled application
+                from inside that container. Requires podman or rootless docker to be installed.
+                If available, 'podman' is preferred and rootless 'docker' is the fallback. Specifying
+                one or the other as '=<container-tool>' forces the use of a specific tool.
+                'dockerfile=<Dockerfile>': Use a user provided 'Dockerfile' instead of the Dockerfile
+                bundled with the application
+
+    --verbose   enable verbose output
+    --help      print this help message
+```
+
+Running the bundled application with the `--with-native-image-agent` argument requires a `native-image-agent` library to be available.
+The output of the `native-image-agent` is written to _<bundle-name>.output/launcher/META-INF/native-image/<bundle-name>-agent_.
+If native image agents output should be inserted into the bundle with `,update-bundle`, the launcher then also requires `native-image`.
+The `update-bundle` option makes executes the command `native-image --bundle-apply=<bundle-name>.nib --bundle-create=<bundle-name>.nib -cp <bundle-name>.output/launcher` after executing the bundled application with the `native-image-agent` attached.
+
+The `container` option realizes a similar behavior to [containerized image builds](#building-in-a-container).
+However, the only exception is that in this case the application is executed inside the container instead of `native-image`.
+Every bundle contains a Dockerfile which is used for executing the bundled application in a container.
+However, this Dockerfile can be overwritten by adding `,dockerile=<path-to-dockerfile>` to the `--container` argument.
```

Comment:
typo: `,dockerile=`

Reply:
Addressed in commit https://github.com/oracle/graal/pull/6496/commits/83db0c17b7812eac5e15f70d23d47ae0fe14cd44